src/error.rs

```diff
@@ -18,6 +20,11 @@ impl Debug for Error {
             Error::DeserializeError => write!(f, "Failed while deserializing data"),
             Error::OpenConfigError => write!(f, "Failed while trying to read config"),
             Error::WriteConfigError => write!(f, "Failed while trying to write data to a file"),
+            Error::NoCorrespondingPathError => write!(
+                f,
+                "Failed while trying to find config file, probably directories doesn't exist."
```

Comment:
small typo here

Reply:
You can commit your own changes on my branch as well